packages/messaging/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/messaging/FirebaseMessagingPlugin.java

```diff
@@ -133,6 +133,7 @@ public void success(String token) {
                         JSObject result = new JSObject();
                         result.put("token", token);
                         call.resolve(result);
+                        handleTokenReceived(token);
```

Comment:
That's not really what I had in mind, since the listener is now called every time you get the token and not just when it has changed. This is a problem because many people use the listener to send the updated token to the server, which can now lead to an unnecessary number of requests. Let me take a quick look at it myself.